tokio/tests/io_sink.rs

```diff
@@ -0,0 +1,18 @@
+#![warn(rust_2018_idioms)]
+#![cfg(all(feature = "full"))]
+
+use tokio::io::AsyncWriteExt;
+
+#[tokio::test]
+async fn sink_is_cooperative() {
+    tokio::select! {
+        biased;
+        _ = async {
+            loop {
+                let buf= vec![1, 2, 3];
+                let _ = tokio::io::sink().write(&buf).await.unwrap();
```

Comment:
I don't think this is ignoring anything?
```suggestion
                tokio::io::sink().write(&buf).await.unwrap();
```

Reply:
Yeah I wrote this way because clippy got angry when I omit let binding because of [unused_io_amount](https://rust-lang.github.io/rust-clippy/master/index.html#/unused_io_amount), but I realized if I use `write_all`, there would be no problem, so I fixed to use it.

https://github.com/tokio-rs/tokio/pull/6254/commits/88d0f639da87b780387454c19e82bb08ea145dbb